<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseWindow</class>
 <widget class="QWidget" name="DatabaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DatabaseWindow</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xueke">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_info">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>XK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>INFO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_insert">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_modify">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_show">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>101</y>
     <width>351</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
